### PR TITLE
Add scripts to display 3.2 code changes

### PR DESCRIPTION
**Pull request by Nicofuma**

### CODE REVIEW

File: src/code-changes/create_diffs.php [hunk 1]
```diff
@@ -353,4 +366,4 @@ function sort_array(&$array)
 fwrite($fh, $output);
 fwrite($fh, $footer);
 fclose($fh);
-?>
\ No newline at end of file
+?>
```
Comment (Crizz0) [inline src/code-changes/create_diffs.php]: Why do we need the closing tag here and not just a empty new line?

File: bin/build_code_changes_32.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+if [ ! -d "$datadir/versions" ];
+then
+	mkdir "$datadir"/versions
+fi
+
+# Clone the repository
+if [ ! -d "$datadir/repo/phpBB" ];
+then
+	git clone git://github.com/phpbb/phpbb3.git "$datadir"/repo
```
Comment (Crizz0) [inline bin/build_code_changes_32.sh]: Should be: `phpbb/phpbb.git` (phpbb3.git is the old name)

File: src/code-changes/create_diffs.php (hunk 1 above)
Comment (Nicofuma) [inline src/code-changes/create_diffs.php]: Because it was already like that :p

File: bin/build_code_changes_32.sh (hunk 2 above)
Comment (Nicofuma) [inline bin/build_code_changes_32.sh]: It just a copy/paste of the other script. We really need to cleanup and few things in this repo

**Comment by Noxwizard** [comment]
`area51/data/code-changes/template/nav_32.html` is missing

**Comment by DavidIQ** [comment]
Ooops... merged the wrong thing...

**Pull request merged**